src/tic_tac_toe/board.clj

```diff
@@ -2,10 +2,30 @@
 
 (def space " ")
 
-(defn init []
-  vector (replicate 9 space))
+(defn get-size [width]
+	(* width width))
+
+(defn get-new-board [width]
+  vector (replicate (get-size width) space))
```

Comment:
Updated. Tested some more and realized i had a misconception on that output of replicate. also changed it to repeat.